cupy/cuda/thrust.pyx

```diff
@@ -53,6 +55,40 @@ cpdef sort(dtype, size_t start, size_t num):
         raise TypeError(msg.format(dtype))
 
 
+cpdef lexsort(dtype, size_t idx_start, size_t keys_start, size_t k, size_t n):
+
+    idx_ptr = <size_t *>idx_start
+    keys_ptr = <void *>keys_start
+
+    # Assuming that Py_ssize_t can be represented with numpy.int64.
+    assert cython.sizeof(Py_ssize_t) == 8
```

Comment:
Please raise exception instead of `assert`, because this can happen depending on the environment.
(I think `NotImplementedError` is appropriate)